cocos/platform/CCPlatformConfig.h

```diff
@@ -110,6 +110,9 @@ THE SOFTWARE.
 #endif
 
 #if (CC_TARGET_PLATFORM == CC_PLATFORM_MAC) || (CC_TARGET_PLATFORM == CC_PLATFORM_IOS)
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_MAC)
+    #define CC_USE_MAC
+#endif
```

Comment:
Don't have need this macro.

Reply:
It uses in TextureMTL.mm, since synchronizeResource only available in macos.